sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/identity.js

```diff
@@ -1,7 +1,7 @@
 var app = angular.module('sentinelDashboardApp');
 
 app.controller('IdentityCtl', ['$scope', '$stateParams', 'IdentityService',
-  'ngDialog', 'FlowServiceV1', 'DegradeService', 'AuthorityRuleService', 'ParamFlowService', 'MachineService',
+  'ngDialog', 'FlowServiceV1', 'DegradeService', 'AuthorityRuleService', 'ParamFlowServiceV2', 'MachineService',
```

Comment:
For the resource chain page, we might need to keep the v1 service. If users are going to push the rules to the config center, they could modify here.